sdk/spring/azure-spring-cloud-trace-sleuth/src/main/java/com/azure/spring/tracing/sleuth/implementation/SpanId.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.tracing.sleuth.implementation;
+
+import javax.annotation.concurrent.Immutable;
+
+import static com.azure.spring.tracing.sleuth.implementation.TraceContextUtil.isValidBase16String;
+
+/**
+ * Helper methods for dealing with a span identifier. A valid span identifier is a 16 character lowercase hex (base16)
+ * String, where at least one of the characters is not a "0".
+ *
+ * <p>There are two more other representation that this class helps with:
+ *
+ * <ul>
+ *   <li>Bytes: a 8-byte array, where valid means that at least one of the bytes is not `\0`.
+ *   <li>Long: a {@code long} value, where valid means that the value is non-zero.
+ * </ul>
+ */
+@Immutable
+public final class SpanId {
```

Comment:
this is no such class within spring-cloud-sleuth project?

Reply:
Yes, not found a suitable class.